edb/graphql/types.py

```diff
@@ -1504,11 +1504,20 @@ def get(self, name: str, *, dummy: bool = False) -> GQLBaseType:
 
             for tname in names:
                 if edb_base is None:
+                    module: Union[s_name.Name, str]
+
                     if '::' in tname:
                         edb_base = self.edb_schema.get(
                             tname,
                             type=s_types.Type,
                         )
+                    elif '__' in tname:
+                        # Looks like it's coming from a specific module
+                        module, tname = tname.split('__', maxsplit=1)
```

Comment:
Will this work right for nested modules?

Reply:
Fixed that to be consistent with nested modules.